demo/writeonly.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * This example overlays a writeonly memfs volume which will effectively "collect" write operations
+ * and leave the underlying volume unchanged. This is useful for test suite scenarios
+ *
+ * Please also see [[../src/__tests__/union.test.ts]] for option examples
+ */
+
+import {ufs} from '../src';
+import {Volume} from 'memfs';
+
+const vol1 = Volume.fromJSON({
+    '/underlying_file': 'hello',
+});
+const vol2 = Volume.fromJSON({});
+
+ufs.use(vol1 as any).use(vol2 as any, {writeonly: true})
```

Comment:
This demo needs to be either updated or removed since `writeonly` is now `writable`.

Reply:
updated